### PR TITLE
Rework layouts

### PR DESCRIPTION
**Pull request by collinjackson**
- Adds back GitHub button
- Makes layout responsive - see screenshots below
- Remove footer text
- Ensure Get Access button doesn't wrap

https://linear.app/capsule-labs/issue/CPSL-814/demo-w-new-branding

Screenshot 2023-11-08 at 6 52 08 PM
Screenshot 2023-11-08 at 6 54 12 PM
Screenshot 2023-11-08 at 6 58 14 PM

### CODE REVIEW

**Comment by nityas** [comment]
A couple of requested visual changes
* buttons -> black might look cleaner, the grey looks a bit odd
* could you add back in the github link? Either in the header or as noted in the designs

Two questions
* there was some progress indicator text in the last version, how does that appear now?
* how does the page handle resizing?

Otherwise looks great!

**Comment by nityas** [comment]
One more thing-- we can remove the disclaimer text on the bottom- recently updated beta env to include a similar warning in the passkey popup

**Comment by collinjackson** [comment]
> * buttons -> black might look cleaner, the grey looks a bit odd

Changed to black but note that the buttons change color to indicate progress.

> * could you add back in the github link? Either in the header or as noted in the designs

Done

> Two questions
> 
> * there was some progress indicator text in the last version, how does that appear now?

Screenshot 2023-11-08 at 5 19 55 PM
Screenshot 2023-11-08 at 5 19 51 PM

File: src/nft-marketplace/pages/home/Home.jsx [hunk 1]
```diff
@@ -11,24 +10,13 @@ const Home = () => {
...
-  </div>
-        <div className="footer">
-        <p>
-          This is a test application to illustrate Capsule's SDK and
-          is intended to be used only for testing purposes. Assets used herein are on
-          the Sepolia testnet and valueless. Additionally, this demo app is connected
-          to a beta environment and test accounts created may be occasionally deleted.
-        </p>
-      </div>
-      </>
```
Comment (vr2amesh) [inline src/nft-marketplace/pages/home/Home.jsx:31]: Are we intending on getting rid of this disclaimer?

File: src/nft-marketplace/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -6,8 +6,9 @@ import { Link } from "react-router-dom";
 
 const Menu = () => (
   <>
+    <Link target="_blank" rel="noreferrer" to='https://github.com/usecapsule/capsule-examples'><p>GitHub</p></Link>
     <Link target="_blank" rel="noreferrer" to='https://docs.usecapsule.com/'><p>Docs</p></Link>
-    <Link target="_blank" rel="noreferrer" to='https://7f4shq8oyfd.typeform.com/to/F86oVLhb'><p class='cta'>Get Access</p></Link>
+    <Link target="_blank" rel="noreferrer" to='https://7f4shq8oyfd.typeform.com/to/F86oVLhb'><p class='cta'>Get&nbsp;Access</p></Link>
```
Comment (vr2amesh) [inline src/nft-marketplace/components/navbar/Navbar.jsx:11]: Good call on the `&nbsp;` to ensure it never gets split over two lines

**Comment by vr2amesh** [comment]
> > * buttons -> black might look cleaner, the grey looks a bit odd
> 
> Changed to black but note that the buttons change color to indicate progress.
> 
> > * could you add back in the github link? Either in the header or as noted in the designs
> 
> Done
> 
> > Two questions
> > 
> > * there was some progress indicator text in the last version, how does that appear now?
> 
> Screenshot 2023-11-08 at 5 19 55 PM Screenshot 2023-11-08 at 5 19 51 PM

Right now, the progress indicator is going to be blue, is this alright? @nityas

**Review by vr2amesh — COMMENTED** [review]
Looks good to me! Much improved layout

File: src/nft-marketplace/pages/home/Home.jsx (hunk 1 above)
Comment (collinjackson) [reply src/nft-marketplace/pages/home/Home.jsx:31]: Yes

**Review by nityas — APPROVED** [review]
looks good!

**Pull request merged**